com.bkmarriott.promotion/src/main/java/com/bkmarriott/promotion/application/schedule/sceduler/CouponIssuancePublishScheduler.java

```diff
@@ -0,0 +1,53 @@
+package com.bkmarriott.promotion.application.schedule.sceduler;
+
+import com.bkmarriott.promotion.application.outputport.CouponExternalEventReader;
+import com.bkmarriott.promotion.application.outputport.CouponExternalEventRecorder;
+import com.bkmarriott.promotion.application.service.CouponExternalEventSendService;
+import com.bkmarriott.promotion.domain.event.CouponIssuanceEvent;
+import com.bkmarriott.promotion.domain.event.DomainEventEnvelop;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class CouponIssuancePublishScheduler {
+
+    private static final long TIME_LIMIT_MINUTES_FOR_OLD = 20;
+
+    private final CouponExternalEventReader eventReader;
+    private final CouponExternalEventRecorder eventRecorder;
+    private final CouponExternalEventSendService eventSendService;
+
+    /**
+     * TIME_LIMIT_MINUTES_FOR_OLD 분 동안 발급되지 못한 엔티티를
+     * 주기적으로 카프카에 발행하는 스케줄러
+     * */
+    @Transactional
+    @Scheduled(fixedRate = 20 * 60000)
```

Comment:
```suggestion
    @Scheduled(fixedRate = TIME_LIMIT_MINUTES_FOR_OLD * 60000)
```
상수를 공통적으로 사용해주면 나중에 관리하기 용이할것 같아요

Reply:
가독성도 올라가고 좋은 방식 같습니다!